crates/matrix-sdk/src/matrix_auth/mod.rs

```diff
@@ -835,6 +850,17 @@ impl MatrixAuth {
 
         Ok(())
     }
+
+    #[cfg(feature = "e2e-encryption")]
+    async fn post_login_cross_signing(&self, auth_data: Option<RumaUiaaAuthData>) {
```

Comment:
This should also be called in place of the manual call to `bootstrap_cross_signing_if_needed` in matrix-sdk/src/oidc/mod.rs

Reply:
Had to move the helper onto `Client` to make that work (without an extra unnecessary `self.client.auth().`), hope that is okay.